src/composables/useFollowSpace.ts

```diff
@@ -27,7 +27,7 @@ export function useFollowSpace(spaceId: any = {}) {
     )
   );
 
-  async function loadFollows() {
+  async function loadFollows(spaceId: string | undefined = undefined) {
```

Comment:
Why do you need `undefined = undefined`?  @ChaituVR

Reply:
Updated 😅